webCommander/centralHub/xTwitter/twitterAPI.py

```diff
@@ -74,10 +88,15 @@ def get_all_tweets_from_user(x_username, max_results=10):
     for tweet in tweets['data']:
         print(tweet['id'])
         print(tweet['text'])
-    return 
+    return
 
 def get_latest_tweet_from_user(x_username):
-    x_user_id = x_userlookup(x_username)['data']['id']
+    try:
+        x_user_id = x_userlookup(x_username)['data']['id']
+    except XUserDoesNotExist as e:
+        # TODO: better handle this
```

Comment:
you can also move this validation up, by changing this function to receive the x_user_id instead of the x_username

Reply:
then you probably also de-duplicate the x_userlookup

Reply:
what's not nice here is that now get_latest_tweet_from_user returns or data or None -> None can cause problems elsewhere (`return` == `return None` in python)